demo/app/src/components/adSize.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Checkbox } from "baseui/checkbox";
+
+const AdSize = props => {
+  const { w, h, checked, setSelection } = props;
```

Comment:
With functional components like this, you can destructure in the param also
`const AdSize = ({ w, h, checked, setSelection }) => (
<Checkbox />
)`

Reply:
Yup, it was broken down just the sake of readability. Now changed to destructure + implicit return.